src/intTest/resources/edifact/pathology.edifact.dat

```diff
@@ -43,6 +43,7 @@ DTM+SRI:201002241541:203'
 GIS+N'
 INV+MQ+42R4.:911::Serum ferritin'
 RSL+NV+11++:::ng/mL'
+STS++FI'
```

Comment:
Shouldn't this be under a `S18+18` segment?

Reply:
Segment group 18 is initiated just by the `GIS+N'` segment - there is no `S18+18` because _reasons_

Reply:
Ahh yes that's fine, forgot when we discussed this together there's no visible segment 18 and we agreed on this